epymodelingsuite/dispatcher/runner.py

```diff
@@ -210,6 +221,56 @@ def run_calibration_with_projection(
         projection_duration = time.time() - projection_start
         logger.info("RUNNER: completed calibration and projection.")
 
+        # Apply anchoring/filtering if specified
+        if configs.anchoring is not None:
+            try:
+                # Load surveillance data
+                surveillance_data = pd.read_csv(configs.anchoring.observed_data_path)
+                
+                # Get distance function
+                distance_function = dist_func_dict[configs.anchoring.distance_function]
+                
+                # Create temporary CalibrationOutput for filtering
+                temp_output = CalibrationOutput(
+                    primary_id=configs.primary_id,
+                    seed=configs.seed,
+                    delta_t=configs.delta_t,
+                    population=configs.model.population.name,
+                    start_date_reference=configs.start_date_reference,
+                    results=projection_results,
+                )
+                
+                # Apply filtering
+                original_count = len(projection_results.projections["baseline"])
+                filtered_projections, filtered_projection_parameters = filter_projections_on_data_point(
```

Comment:
The latest function name defined in filtering.py is `anchor_projections_on_data()`. Also needs import update.